src/lightbox/LightboxActionSheet.js

```diff
@@ -7,31 +7,31 @@ import Toast from '../utils/showToast';
 
 type DownloadImageType = {
   url: string,
-  auth: Auth,
+  auth: Auth
 };
 
 type ShareLinkType = {
-  url: string,
+  url: string
 };
 
 type ExecuteActionSheetActionType = {
   title: string,
   url: string,
-  auth: Auth,
+  auth: Auth
 };
 
 type ButtonProps = {
   auth?: Auth,
-  url: string,
+  url: string
 };
 
 type ButtonType = {
   title: string,
-  onPress: (props: ButtonProps) => void | boolean,
+  onPress: (props: ButtonProps) => void | boolean
```

Comment:
I think your `prettier` is configured inconsistently with the rest.

Reply:
Not able to get which attribute is configured wrongly 🤔